tapfiliate/tapfiliate.php

```diff
@@ -8,6 +8,8 @@ class Tapfiliate extends Module
 {
     const UPDATE_TYPE_UPDATE = 'order_update';
     const UPDATE_TYPE_NEW = 'order_new';
+    // @TODO: Production url
+    const TAPFILIATE_BASE_URL = 'https://app.dev.tap';
```

Comment:
Use Prod host

Reply:
later on it would be required, first we should test it on test environment with test URL at least...